libqtile/manager.py

```diff
@@ -1103,7 +1102,12 @@ def _items(self, name):
         elif name == "widget":
             return False, list(self.widgetMap.keys())
         elif name == "bar":
-            return False, [x.position for x in self.currentScreen.gaps]
+            keys = []
+            for gap in self.currentScreen.gaps:
+                position = getattr(self.currentScreen, gap.position)
+                index = position.index(gap)
+                keys.append("{}_{}".format(gap.position, index))
```

Comment:
I wonder if we can make "bottom" an alias? or have it be a list of gaps instead of bottom_0?

Reply:
As I said in the original description, I do agree that it's not perfect but I couldn't manage to do this without rewriting the whole `CommandTree` logic.